### PR TITLE
fix timing issue with buttons in hydrator details

### PR DESCRIPTION
**Pull request by elfenheart**
details

### CODE REVIEW

File: cdap-ui/app/features/adapters/templates/detail.html [hunk 1]
```diff
@@ -59,36 +58,51 @@
     <div class="btn btn-cdap"
       ui-sref="apps.detail.overview.status({ appId: app.name })">
       View in CDAP
-    </div
+    </div>
 
     <!-- IF ETL REALTIME -->
-    <div class="btn"
-      ng-click="do('Start')"
+    <div class="pull-left"
```
Comment (astewes) [inline cdap-ui/app/features/adapters/templates/detail.html]: Very minor, but let's remove this empty line in between attributes

**Comment by astewes** [comment]
One minor comment but otherwise LGTM

**Comment by elfenheart** [comment]
@astewes comments addressed

**Comment by ajainarayanan** [comment]
:+1: LGTM

**Pull request merged**